packages/dev/core/src/Gizmos/planeRotationGizmo.ts

```diff
@@ -122,34 +122,37 @@ export class PlaneRotationGizmo extends Gizmo implements IPlaneRotationGizmo {
             vUV = uv;
         }`;
 
-    protected static _RotationGizmoFragmentShader = `
-        precision highp float;
-        varying vec2 vUV;
-        varying vec3 vPosition;
-        uniform vec3 angles;
-        #define twopi 6.283185307
-        void main(void) {
-            vec2 uv = vUV - vec2(0.5);
-            float angle = atan(uv.y, uv.x) + 3.141592;
-            float delta = gl_FrontFacing ? angles.y : -angles.y;
-            float begin = angles.x - delta * angles.z;
-            float start = (begin < (begin + delta)) ? begin : (begin + delta);
-            float end = (begin > (begin + delta)) ? begin : (begin + delta);
-            float len = sqrt(dot(uv,uv));
-            float opacity = 1. - step(0.5, len);
-
-            float base = abs(floor(start / twopi)) * twopi;
-            start += base;
-            end += base;
-
-            float intensity = 0.;
-            for (int i = 0; i < 5; i++)
-            {
-                intensity += max(step(start, angle) - step(end, angle), 0.);
-                angle += twopi;
+    protected static _createRotationGizmoFragmentShader(color: Color3) {

```

Comment:
Technically it is a breaking change if someone subclassed this class and replaced `_RotationGizmoFragmentShader`...

To be safe, you could do something like:
```typescript
protected static _RotationGizmoFragmentShader: Nullable<string> = null;
...
Effect.ShadersStore["rotationGizmoFragmentShader"] = PlaneRotationGizmo._RotationGizmoFragmentShader ?? PlaneRotationGizmo._createRotationGizmoFragmentShader(hoverColor);
```
But I'm not sure how many people would have subclassed `PlaneRotationGizmo` in the first place...